Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Health Goes Down With Age

## Changelog:
- Once again, just like in the vanilla game, health goes down with age allowing characters to die of old age.

# Tasks:
- [x] Check health of new spawned characters


# How to test:
Run the game for a few decades, confirm humans die of old age like in the vanilla game around 70-50 years.

On the other hand, elves and other long-lived races shouldn't die of old age when they're just 150 years old.
